tests/test_checkout.py

```diff
@@ -208,14 +210,17 @@ def test(self):
         ret = main(["config", "cache.type", "copy"])
         self.assertEqual(ret, 0)
 
-        stages = self.dvc.add(self.DATA_DIR)
-        self.assertEqual(len(stages), 1)
-        stage = stages[0]
+        # stages = self.dvc.add(self.DATA_DIR)
```

Comment:
If you don't need this one, please remove it :)

Reply:
sorry about that